Chap_API_Fabric.tex

```diff
@@ -421,10 +421,10 @@ \subsection{Fabric Operation Constants}
 
 \begin{constantdesc}
 %
-\declareconstitemNEW{PMIX_FABRIC_REQUEST_INFO}
+\declareconstitemvalueNEW{PMIX_FABRIC_REQUEST_INFO}{0}
```

Comment:
These values appear to only be used between library and RM?  If so, we could decide we don't want to specify them?

Reply:
I think we have to retain that specification so the RM can also remain implementation agnostic (e.g., should the RM want to dlopen/dlsym the PMIx symbols)